### ISSUE TITLE
Line-by-line navigation is not working well for JAWS in Chrome

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#7337]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.535
    OS Version      : Windows 10 (with WSL2), JAWS 2020.2006.12.400, and Chrome (83.0.4103.116)
    R Version       : any

### Steps to reproduce the problem

1. Enable "Screen Reader Support."
2. Press Ctrl+1 to focus in source editor.
3. Type any two lines. For example:

``` r
First line
Second line
```

4.Put your cursor to the very first position of the first line by pressing home key.
5. Press down-arrow key to go to the next line and up-arrow key to come back, and see if JAWS announces those lines.

### Describe the problem in detail

JAWS does not read aloud any lines when user's system cursor is positioned at a certain column. As far as I have tested, JAWS only announces correctly when the caret position is at the very end of each line. Thus, users manually have to put their cursor to the end by pressing "end" key.

### Describe the behavior you expected

JAWS must announce each line regardless of any system caret position.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Confirmed the repro on 1.3 and 1.4. Compare with behavior of Visual Studio Code editor as an example the behavior we want here.

**Comment by gtritchie** [comment]
Don't expect to get this done in 1.4, too risky to touch core Ace code at this stage, so moving to 1.5.

**Comment by gtritchie** [comment]
This can wait until after the Electron switch, moving to 1.6.

**Comment by gtritchie** [comment]
This seems significantly improved since I last tested it. There are still some problems such as if you have multiple blank lines, and you navigate through them with up/down arrows, only the first blank line encountered is announced. Nothing is said for the others.

**Comment by gtritchie** [comment]
Though sometimes I stop getting any announcements from JAWS when moving around in the editor. I need to play with this more to figure out what might be causing that!

**Comment by gtritchie** [comment]
Now that we've updated to a current Ace release and are setting the new keyboard accessibility flag, the editor is behaving better with JAWS and NVDA in my testing.

**Comment by jonvanausdeln** [comment]
Verified on https://dailies.rstudio.com/version/2023.12.0-daily+262
As @gtritchie mentioned, this appears to be acting more as expected.
